library/src/test/java/com/chuckerteam/chucker/internal/support/FormatUtilsTest.kt

```diff
@@ -1,18 +1,17 @@
 package com.chuckerteam.chucker.internal.support
 
 import com.google.common.truth.Truth.assertThat
-import org.junit.jupiter.api.Test
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
 
+@RunWith(RobolectricTestRunner::class)
```

Comment:
Why would we add the whole abandoned Roboelectic to just have `JSONObject`?
Why don't you just add dependency to json directly instead of it?
Like `testImplementation 'org.json:json:20190722'`

Reply:
Removed robolectric dependency and used moshi for formatting json.

Also, found something interesting, there are API differences between android's `org.json` and the original `org.json:json:20190722`.

I tried adding `testImplementation 'org.json:json:20190722'` and found that the function `JSONObject(json).toString(indentSpaces)` works differently on these two APIs.

For example,

For json string
```
{ "field" : null }
```
android's `org.json` will return
```
{
  "field": null
}
```
but, the original `org.json:json:20190722` will return
```
{"field": null}
```